src/odemis/gui/cont/acquisition/cryo_acq.py

```diff
@@ -171,9 +180,14 @@ def __init__(self, tab_data, panel, tab):
         self._tab_data.main.features.subscribe(self._on_features_change, init=True)
 
         # advanced features toggle
+        self._panel.fp_automation.Show(ODEMIS_ADVANCED_FLAG)
         self._panel.btn_acquire_features.Show(ODEMIS_ADVANCED_FLAG)
         self._panel.chk_use_autofocus_acquire_features.Show(ODEMIS_ADVANCED_FLAG)
 
+        # connect features to chklistbox
```

Comment:
```suggestion
        # connect features to checklist
```
I see that you use "chk" at other places, but maybe nicer to write in full in comment